docs/Downloaders/SABnzbd/Basic-Setup.md

```diff
@@ -42,13 +42,16 @@ Here you setup your download path/location.
 `Settings` => `Folders` => `System Folders`
 ![!Folders: System Folders](images/sabnzbd-folders-system-folders.png)
 
-I recommend the .nzb Backup Folder because sabnzbd doesn't have a hidden history and having remove from history enabled in Sonarr/Radarr is the best method. This allows you to see your past downloads and it is used for duplicate download detection/prevention. Default is empty, I picked history because it is easy. It'll end up in the `/config` folder for Docker, which isn't crazy... but this is literally compressed nzb files, so can end up pretty big. But the choice is yours what you prefer.
+!!! info "Starting from 4.3.x+ SABnzbd has a hidden (archive) history."
+
+It's still recommended to use the .nzb Backup Folder its still useful for dupe detection (hash matching) or just need to retry something from the past.
+Default is empty, I picked history because it is easy. It'll end up in the `/config` folder for Docker, which isn't crazy... but this is literally compressed nzb files, so can end up pretty big. But the choice is yours what you prefer.
```

Comment:
```suggestion
Using the .nzb Backup Folder is still recommended as it is useful for dupe detection (hash matching) or if you need to retry something from the past.
The default is empty, I picked history because it is easy. It'll end up in the `/config` folder for Docker, which isn't crazy... but this is only compressed nzb files, so it can end up pretty big. The choice is yours what you prefer.